bigchaindb/assets.py

```diff
@@ -0,0 +1,78 @@
+import rethinkdb as r
+
+from bigchaindb.exceptions import AssetIdMismatch, TransactionDoesNotExist, AmountError
+
+
+def get_asset_id(txids, bigchain):
+    """Get the asset id from a list of transaction ids.
+
+    This is useful when we want to check if the multiple inputs of a transaction
+    are related to the same asset id.
+
+    Args:
+        txids (list): list of transaction ids.
+
+    Returns:
+        str: uuid of the asset.
+
+    Raises:
+        AssetIdMismatch: If the inputs are related to different assets.
+    """
+
+    if not isinstance(txids, list):
+        txids = [txids]
+
+    asset_ids = []
+    for txid in set(txids):
+        tx = bigchain.get_transaction(txid)
+        if tx is None:
+            raise TransactionDoesNotExist('Transaction with txid `{}` does not exist in the bigchain'.format(txid))
+        asset_ids.append(tx['transaction']['asset']['id'])
+
+    asset_ids = set(asset_ids)
+    if len(asset_ids) > 1:
+        raise AssetIdMismatch("All inputs of a transaction need to have the same asset id.")
+    return asset_ids.pop()
+
+
+def validate_asset_creation(asset_data, divisible, updatable, refillable, amount):
```

Comment:
Should all these parameters default to `None`? Maybe make everything after `asset_data` keyword only?